src/dfi/consensus/tokens.cpp

```diff
@@ -64,23 +64,20 @@ ResVal<CScript> CTokensConsensus::MintableToken(DCT_ID id,
         return Res::Err("token %s is not mintable!", id.ToString());
     }
 
-    ResVal<CScript> result = {auth.out.scriptPubKey, Res::Ok()};
+    ResVal<CScript> result{auth.out.scriptPubKey, Res::Ok()};
     if (anybodyCanMint || HasAuth(auth.out.scriptPubKey)) {
         return result;
     }
 
-    // Historic: in the case of DAT, it's ok to do not check foundation auth cause exact DAT owner is foundation
-    // member himself The above is no longer true.
-
-    if (token.IsDAT()) {
-        // Is a DAT, check founders auth
-        if (height < static_cast<uint32_t>(consensus.DF20GrandCentralHeight) && !HasFoundationAuth()) {
-            return Res::Err("token is DAT and tx not from foundation member");
-        }
-    } else {
+    if (!token.IsDAT()) {
         return Res::Err("tx must have at least one input from token owner");
     }
 
+    // It is a DAT, check founders auth
```

Comment:
Is the reason to drop the https://github.com/DeFiCh/ain/pull/2730/files#diff-48843c84274310df9b298a031159a1961a0b0cdb4e85b2601def1305bee1febaL77 check intentional?

Reply:
Yes. GrandCentral introduced consortium, so that if a DAT token without foundation auth did not fail at this point as we would then check for a consortium member. Now we are removing consortium we restore the previous behaviour, simply that, a none-DAT token must have an input from the token owner and a DAT token must have HasFoundationAuth.

Refer to the code linked below.

https://github.com/DeFiCh/ain/blob/17c7d0085c13a21e78e368b1c808299c4dfabf85/src/masternodes/mn_checks.cpp#L773-L779